hugr-core/src/extension/op_def.rs

```diff
@@ -557,12 +575,14 @@ pub(super) mod test {
                 // a compile error here. To fix: modify the fields matched on here,
                 // maintaining the lack of `..` and, for each part that is
                 // serializable, ensure we are checking it for equality below.
-                SignatureFunc::PolyFuncType(CustomValidator {
+                SignatureFunc::CustomValidator(CustomValidator {
                     poly_func,
                     validate: _,
-                }) => Some(poly_func.clone()),
+                })
+                | SignatureFunc::PolyFuncType(poly_func)
+                | SignatureFunc::MissingValidateFunc(poly_func) => Some(poly_func.clone()),
                 // This is ruled out by `new()` but leave it here for later.
-                SignatureFunc::CustomFunc(_) => None,
+                SignatureFunc::CustomFunc(_) | SignatureFunc::MissingComputeFunc => None,
```

Comment:
I suspect the comment on the previous line can go (I suspect "later" meant "when we have implemented serialization", essentially!), please clarify if I'm wrong